packages/contentstack-seed/package.json

```diff
@@ -8,7 +8,6 @@
     "@contentstack/cli-cm-import": "~1.8.4",
```

Comment:
@vkalta Can you please bump the package version?

Reply:
@vkalta Package lock is not updated with the new version bump, can you please generate a new package-lock and push it?